### DISCUSSION TITLE
Running Custom QT Wayland Compositor using WSLg

### DISCUSSION THREAD

**Discussion by marshalltesseract** [microsoft/wslg#1069] [Q&A]
Very new to the world of WSL. I have a custom QT Wayland Compositor I am working on for a touchscreen-centric device to allow it to run a bunch of Wayland apps inside a touch-friendly compositor. I was wondering if it was possible to run a Wayland Compositor using WSLg the same way someone can run a Wayland application using WSLg.

If not, what are alternatives/stack I would have to use?

Cheers!

**Comment by ericblade** [comment]
also looking for this... but so far, i haven't yet figured out how to run Qt Wayland apps in WSLg yet... just getting seg faults so far.
